Build.mak

```diff
@@ -26,4 +26,10 @@ $B/overview: $O/libvibed.a $O/libdyaml.a
 $B/overview: $C/src/overview/main.d
 $B/overview: override LDFLAGS += -L$O -lvibed -ldyaml -levent -lssl -lcrypto
 
-all += $B/overview
+$B/neptune: override LDFLAGS += -L$O -lvibed -levent -lssl -lcrypto
+$B/neptune: $O/libvibed.a
+$B/neptune: $C/src/neptune/main.d
+
+neptune: $B/neptune
```

Comment:
You seem to keep calling the tool "neptune" - initially I thought it is matter of keeping things simple in communication but it propagated into actual PR. This is not correct - neptune is a name for the whole set of rules and tools defined by this repo. Your PR implements just own such tool, "release neptune library".

I would suggest calling individual binaries like `neptune-overview`, `neptune-releaselib` and so in.